### PR TITLE
Fix #993 correctly rewrite referenced file paths in CSS files

### LINKED ISSUE #993: Incorrect path for files referenced in minified CSS if WordPress is in subfolder
Since 3.0.4, path for files referenced in minified CSS are incorrectly rewritten if WordPress is in a subfolder.

This is because by default `ABSPATH` is used as the root path, but in a subfolder configuration, `ABSPATH` returns the path to the subfolder instead of the root path.

This also applis to configuration where `wp-content` default path has been changed.

### PR DESCRIPTION
**Pull request by Tabrisrp**
@Screenfeed do you see any problem with using `SCRIPT_FILENAME` here?

### CODE REVIEW

File: inc/functions/files.php [hunk 1]
```diff
@@ -1091,7 +1091,7 @@ function rocket_fetch_and_cache_busting( $src, $cache_busting_paths, $abspath_sr
 		 *
 		 * @param string The Document Root path.
 		*/
-		$document_root = apply_filters( 'rocket_min_documentRoot', ABSPATH );
+		$document_root = apply_filters( 'rocket_min_documentRoot', wp_normalize_path( dirname( $_SERVER['SCRIPT_FILENAME'] ) ) );
```
Comment (Screenfeed) [inline inc/functions/files.php]: While `ABSPATH` has a trailing slash, `dirname()` will remove it from `$_SERVER['SCRIPT_FILENAME']`. Is that an issue?

Comment (Tabrisrp) [reply inc/functions/files.php]: The default value used in the Minify_CSS_Rewriter class is `$_SERVER['DOCUMENT_ROOT']` which doesn't have a trailing slash either, so it's fine to not have it either here

**Comment by Tabrisrp** [comment]
good to merge?

**Comment by Screenfeed** [comment]
I don't see what you're talking about 😛

**Pull request merged**